modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -442,6 +442,22 @@ Generated schema location: `+http://www.mulesoft.org/schema/mule/hello-prefix/cu
 
 Generated schema location is `+http://www.mulesoft.org/schema/mule/hello-with-spaces/current/mule-hello-with-spaces.xsd+`
 
+[[prefix-attribute]]
+=== Prefix Attribute
+
+The `prefix` attribute in the `<module>` element is the prefix of the module
```

Comment:
Suggestion for consistency with the text above, start with "The prefix of the module used when generating schemas."